boards/nucleo-f334r8/include/periph_conf.h

```diff
@@ -137,10 +169,10 @@ static const uint8_t spi_divtable[2][5] = {
     },
     {       /* for APB2 @ 72000000Hz */
         7,  /* -> 281250Hz */
-        7,  /* -> 281250Hz */
+        6,  /* -> 562500Hz */
         5,  /* -> 1125000Hz */
-        3,  /* -> 4500000Hz */
-        2   /* -> 9000000Hz */
+        2,  /* -> 9000000Hz */
+        1   /* -> 18000000Hz */
```

Comment:
The divtable changes are out of the scope of this PR and are wrong IMHO. This value should correspond to a 10MHz SPI clock speed so the initial 9MHz was close enough and should not be changed. (note that there's work in #14749 to improve the situation)